### PR TITLE
Scaladoc: Fix double implicit modifier

### LINKED ISSUE #13869: Scaladoc: Some implicit classes are too implicit
### Description
Some implicit classes have too many implicit modifiers.
image
Location: [Predef](https://dotty.epfl.ch/api/scala/Predef$.html)
### Expectation
Just one implicit modifier

### PR DESCRIPTION
**Pull request by pikinier20**
closes #13869 

It seems that compiler now adds Implicit flag to both desugarized method and class. (Before only method was implicit)

### CODE REVIEW

File: scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala [hunk 1]
```diff
@@ -100,7 +100,7 @@ object SymOps:
         Flags.Case -> Modifier.Case,
       ).collect {
         case (flag, mod) if sym.flags.is(flag) => mod
-      } ++ Seq(Modifier.Implicit).filter(_ => sym.isImplicitClass)
```
Comment (KacperFKorban) [inline scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala:103]: Is `isImplicitClass` used anywhere else?

**Comment by julienrf** [comment]
Would it be possible to cover this with tests?

File: scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala (hunk 1 above)
Comment (pikinier20) [reply scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala:103]: It's used in InkuireSupport :)

**Comment by pikinier20** [comment]
> Would it be possible to cover this with tests?

Added

**Review by BarkingBad — APPROVED** [review]
obraz

Looks good now

File: scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala (hunk 1 above)
Comment (KacperFKorban) [reply scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala:103]: Ok, can that be rewritten as well now?

**Pull request merged**